src/main/java/pl/edu/pw/pik/pikactivitytrackerserver/service/WebsiteService.java

```diff
@@ -1,21 +1,30 @@
 package pl.edu.pw.pik.pikactivitytrackerserver.service;
 
+
+import com.mongodb.client.MongoCollection;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.actuate.autoconfigure.metrics.MetricsProperties;
+import org.springframework.data.mongodb.core.mapping.Document;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.PathVariable;
 import pl.edu.pw.pik.pikactivitytrackerserver.DTO.WebsiteDTO;
 import pl.edu.pw.pik.pikactivitytrackerserver.Repository.WebsitesRepository;
 import pl.edu.pw.pik.pikactivitytrackerserver.model.Website;
+
 import java.util.List;
 import java.util.UUID;
 
+//it means that this class contains buisness logic
 @Service
 @Transactional
 public class WebsiteService {
```

Comment:
collection Dal chyba nie jest zaimportowane(u mnie jak ściągnęłam to nie było)

Reply:
naprawione